go/vt/sqlparser/normalizer_test.go

```diff
@@ -1322,9 +1318,9 @@ JOIN warehouse%d AS w ON c_w_id=w_id
 WHERE w_id = %d
 AND c_d_id = %d
 AND c_id = %d`,
-		`SELECT d_next_o_id, d_tax 
-FROM district%d 
-WHERE d_w_id = %d 
+		`SELECT d_next_o_id, d_tax
```

Comment:
Hmm, my IDE didn't even show these changes going out, let me see if the tests break...